### PR TITLE
added odo extension to overview

### PR DESCRIPTION
**Pull request by j-c-berger**
Signed-off-by: Jacob Berger <jacob.berger@ibm.com>

In response to issue #646 `Codewind for Eclipse Che Overview Section Enhancement`: https://github.com/eclipse/codewind/issues/646 

I added the instructions for the ODO extension to the overview.

### CODE REVIEW

**Comment by micgibso** [comment]
Ready for review.  @GeekArthur

File: docs/_documentations/mdt-che-overview.md [hunk 1]
```diff
@@ -21,3 +21,24 @@ The Eclipse tools are [open source](https://github.com/eclipse/codewind-eclipse)
...
+### Add additional rules to support Codewind ODO extension
+The ODO extension needs to add additional rules for accessing OpenShift resources:
+1. In your home directory, run the following command to clone the ODO extension repository:
+`git clone https://github.com/eclipse/codewind-odo-extension`
+2. Login to your OpenShift/OKD cluster
+3. Go into `~/codewind-odo-extension/odo-RBAC` then run the following commands to add additional rules:
+`kubectl apply -f codewind-odoclusterrole.yaml`
+`kubectl apply -f codewind-odoclusterrolebinding.yaml`
+
+### Install Codewind ODO extension manually
```
Comment (elsony) [inline docs/_documentations/mdt-che-overview.md]: @GeekArthur Is there any reason why we need to include manually install odo extension info in the doc? Don't we always install it automatically for user?

File: docs/_documentations/installoncloud.md [hunk 2]
```diff
@@ -10,129 +10,35 @@ order: 20
...
-### Building a project 
-Go to **View**>**Find Command…**>**Codewind: Build**.
-
-- Alternative instructions:
-  - Enter the following command: `curl -k -H "Content-Type: application/json" -X POST https://codewind-release:9191/api/v1/projects/8801a6d0-7805-11e9-b22f-19482c5ffbd6/build -d '{"action": "build"}'`
-
-## Updating the version
-Restart the Codewind workspace in Che. Che automatically pulls the newest version of Codewind and the Theia extension.
\ No newline at end of file
+### Install Codewind ODO extension manually
```
[the remark this answers is not in the document]
Comment (j-c-berger) [reply docs/_documentations/installoncloud.md]: @elsony would you like me to delete this section then?

File: docs/_documentations/mdt-che-overview.md (hunk 1 above)
Comment (GeekArthur) [reply docs/_documentations/mdt-che-overview.md]: @elsony It’s only for installer doesn’t work properly, in order to not block user use the ODO extension we provide manual install

File: docs/_documentations/installoncloud.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (GeekArthur) [reply docs/_documentations/installoncloud.md]: @elsony It’s only for installer doesn’t work properly, in order to not block user use the ODO extension we provide manual install

[the remark this answers is not in the document]
Comment (GeekArthur) [reply docs/_documentations/installoncloud.md]: @elsony But for landing page, I think we can move the **manual install** to trouble shooting page for user hit the auto install doesn't work properly issue. And don't include **manual install** in our main `Codewind for Eclipse Che Overview Section`

File: docs/_documentations/mdt-che-overview.md (hunk 1 above)
Comment (GeekArthur) [reply docs/_documentations/mdt-che-overview.md]: @elsony But for landing page, I think we can move the manual install to trouble shooting page for user hit the auto install doesn't work properly issue. And don't include manual install in our main Codewind for Eclipse Che Overview Section

**Comment by j-c-berger** [comment]
@elsony @GeekArthur I made the suggested changes. PR is now up to date. Thanks!

**Comment by GeekArthur** [comment]
@elsony Talked with Jake, we also need to remove `manual install` in docs/_documentations/mdt-che-overview.md as you suggest then we are all good.

**Comment by j-c-berger** [comment]
I now removed the instructions to manually install from `mdt-che-overview.md` as well

**Comment by GeekArthur** [comment]
Looks good to me. Thanks.

**Comment by sishida** [comment]
@elsony Hi, Elson, Jingfu and I have discussed this issue. Could you please add your approval as a committer so that we can merge? Thank you!

**Pull request merged**